tests/components/wemo/test_binary_sensor.py

```diff
@@ -0,0 +1,18 @@
+"""Tests for Belkin WeMo binary_sensor."""
+import pytest
+
+from homeassistant.components.wemo import binary_sensor
+
+from . import ConfigEntryTests, DeviceTests, SubscriptionTests
+
+
+@pytest.fixture
+def wemo_device(pywemo_device):
+    """Create a device for the WemoBinarySensor."""
+    return binary_sensor.WemoBinarySensor(pywemo_device)
```

Comment:
Don't create the entity directly. Set up the integration and assert the entity state from the state machine.